lib/dealer.rb

```diff
@@ -0,0 +1,31 @@
+class Dealer
+
+  attr_reader :attributes
+
+  def initialize(attributes)
+    @attributes = attributes
+  end
+
+  def new_deck
+    cards.shuffle
+  end
+
+  def get_card(id)
+    cards[id]
```

Comment:
AHA, you're shuffling an instance of cards to pass the Game for it's record of what the deck is, but the order never changes in here. so an array index lookup is equivalent to an id lookup. -> rather confusing. While it works, I'd love to see you make use of the `id` on Card with a detect lookup.